arrow/arrow.py

```diff
@@ -969,8 +969,50 @@ def humanize(
                 )
             )
 
-    # query functions
+    def dehumanize(self, humanized_str="now", locale="en_us"):
+        """reads a localized, humanized representation of a relative difference in time,
+        returns a metadata of time.
```

Comment:
I would change this to say it returns an Arrow object. You can look at other function docstrings for motivation on what to include :).